qiskit/algorithms/minimum_eigen_solvers/vqe.py

```diff
@@ -133,6 +135,8 @@ def __init__(
                 These are: the evaluation count, the optimizer parameters for the
                 ansatz, the evaluated mean and the evaluated standard deviation.`
             quantum_instance: Quantum Instance or Backend
+            split_transpile: ``True`` allows transpiling circuits for ansatz and expectation
```

Comment:
If we go with this approach we should probably add the same flag to QAOA, which is implemented as a sub-class of VQE, so we can leverage the same improvement when using that algorithm too.